tests/e2e/constants/BASE_TEST_CONSTANTS.ts

```diff
@@ -145,6 +146,12 @@ export const BASE_TEST_CONSTANTS: {
 	SELECT_OPENING_EXISTING_WORKSPACE_INSTEAD_OF_CREATION_NEW:
 		process.env.SELECT_OPENING_EXISTING_WORKSPACE_INSTEAD_OF_CREATION_NEW === 'true',
 
+	/**
+	 * select creation of a new workspace, if a duplicate workspace is created from the factory or sample list.
+	 * this option is false by default.
+	 */
+	SELECT_OF_CREATION_NEW_WORKSPACE: process.env.SELECT_OF_CREATION_NEW_WORKSPACE === 'true',
```

Comment:
IMHO `SELECT_CREATING_NEW_WORKSPACE` name should be more relevant in this case

Reply:
Also, default value = true is probably conflicting with another constant value `SELECT_OPENING_EXISTING_WORKSPACE_INSTEAD_OF_CREATION_NEW=== true`

Reply:
Both constants `SELECT_CREATING_NEW_WORKSPACE` and `SELECT_OPENING_EXISTING_WORKSPACE_INSTEAD_OF_CREATION_NEW`  have default value `false`